server/mdm/microsoft/microsoft_mdm.go

```diff
@@ -396,8 +396,8 @@ const (
 )
 
 const (
-	FleetBitLockerTargetLocURI = "./Device/Vendor/MSFT/BitLocker"
-	FleetOSUpdateTargetLocURI  = "./Device/Vendor/MSFT/Policy/Config/Update"
+	FleetBitLockerTargetLocURI = "/Vendor/MSFT/BitLocker"
+	FleetOSUpdateTargetLocURI  = "/Vendor/MSFT/Policy/Config/Update"
```

Comment:
Again just for my own understanding, is it so we also block user bitlocker/policy settings?

Reply:
exactly! this is because you can omit the `./Device` part if you want:

> For device wide configuration the Device/ portion may be omitted from the path, deeming the following paths respectively equivalent to the paths provided above:
>
>    ./Vendor/MSFT/Policy/Config/AreaName/PolicyName to configure the policy.
>    ./Vendor/MSFT/Policy/Result/AreaName/PolicyName to get the result.
>
> _Source: https://learn.microsoft.com/en-us/windows/client-management/mdm/policy-configuration-service-provider#policy-scope_